zlib.h.in

```diff
@@ -62,7 +62,7 @@ extern "C" {
 #define ZLIB_VER_MAJOR 1
 #define ZLIB_VER_MINOR 2
 #define ZLIB_VER_REVISION 12
-#define ZLIB_VER_SUBREVISION 0
+#define ZLIB_VER_SUBREVISION 0xf
```

Comment:
This breaks the pattern, as all other numbers are decimal, and this is hexadecimal. We really don't need to care about what Mark Adler thinks as they are numerically equivalent. We can't say the same about strings, as clearly `.f` and `.15` are not the same string.

Reply:
It does match visually with the version used elsewhere and is more easily an indication of the fact that it is f for fork.